pyart/retrieve/comp_z.py

```diff
@@ -0,0 +1,109 @@
+"""
+Calculate the composite reflectivity 
+
+"""
+
+import copy
+import numpy as np
+from netCDF4 import num2date
+from pandas import to_datetime
+
+def composite_reflectivity(radar,rhv_filter=True,rhv_value=0.95,verbose=False):
+    
+    """
+    Composite Reflectivity 
+    
+    Often a legacy product, composite reflectivity is: 
+    "A display or mapping of the maximum radar reflectivity factor at any altitude as a function of position on the ground." - AMS Glossary 
```

Comment:
Can you add citation here?

Reply:
On second thought, I don't see a well-defined (or citable) definition out there...